docs/framework/unmanaged-api/profiling/icorprofilerinfo10-enumerateobjectreferences-method.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "ICorProfilerInfo10::EnumerateObjectReferences"
+ms.date: "08/06/2019"
+dev_langs: 
+  - "cpp"
+api_name: 
+  - "ICorProfilerInfo10.EnumerateObjectReferences"
+api_location: 
+  - "mscorwks.dll"
+api_type: 
+  - "COM"
+author: "davmason"
+ms.author: "davmason"
+---
+# ICorProfilerInfo10::EnumerateObjectReferences Method
+  
+ Given an ObjectID, callback and clientData, enumerates each object reference (if any).   
+  
+## Syntax  
+  
+```cpp
+HRESULT EnumerateObjectReferences( [in] ObjectID objectId,
+                                   [in] ObjectReferenceCallback callback,
+                                   [in] void* clientData);
+```  
+  
+#### Parameters  
+
+ `objectId`
```

Comment:
```suggestion
 `objectId` \
```
We've been following the convention of having the parameter description on a separate line. That requires either two spaces or a ` \` at the end of a line if there's no following blank line.